src/ecosystem/Ecosystem.jl

```diff
@@ -7,11 +7,26 @@ using Distributions
 
 abstract type EcoModel end
 
+const DISCRETE_FACTOR_TYPES = ["integer", "categorical"]
+
+"""
+Check ptype for discrete variable types.
+
+Returns true if discrete, false otherwise.
+
+# Arguments
+- `ptype` : String representing variable type.
+"""
```

Comment:
Fix docstring structure

```suggestion
"""
    _check_discrete(p_type::String)::Bool

Check ptype for discrete variable types.
Returns true if discrete, false otherwise.

# Arguments
- `ptype` : String representing variable type.
"""
```